designs/PackageVulnerability/DotnetListPackageVulnerable.md

```diff
@@ -165,42 +140,43 @@ The following sources were used:
 
 Package                Requested   Resolved    Latest
 EntityFramework        6.1.2       6.1.2       6.2.0
-NUnit                  2.4.0 (D)   2.6.4       3.8.1  
-My.Sample.Pkg          2.1.3 (V)   4.1.0       4.1.0
+NUnit                  2.4.0       2.6.4 (D)   3.8.1  
+My.Sample.Pkg          2.1.3       4.1.0 (V)   4.1.0
 
+ > To see transitive packages, add the `--include-transitive` option.
 (D): Deprecated package(s). Use 'dotnet list package --deprecated' for more info.
 (V): Vulnerable package(s). Use 'dotnet list package --vulnerable' for more info.
 ```
+Notes:
+* *Important! Installed packages that are vulnerable or deprecated but not outdated will not be part of this command's output.*
+
+* If any of the listed packages is deprecated, a `(D)` marker is printed next to the version indicating the package version is deprecated. A description is printed to the console explaining what the `(D)` marker means, but only if the `(D)` marker is used.
 
-*Combining the `--outdated` option with the `--offline` options is not supported. When doing so, the following error message will be presented:*
+* If any of the listed packages is vulnerable, a `(V)` marker is printed next to the version indicating the package version is known to be vulnerable. A description is printed to the console explaining what the `(V)` marker means, but only if the `(V)` marker is used.
+
+*Combining the `--vulnerable` option with the `--outdated` option is (currently) not supported. When doing so, the following error message will be presented:*
```

Comment:
Just curious, why don't we support combining these options? Engineering cost?

Reply:
The main reason would be that the format of the reports changes from type to type --the columns we see in `--vulnerable` are different from `--outdated` etc. If we wanted to combine them, we could--it wouldn't be a huge engineering cost--we'd just have to decide how to format the output. It might be a good question to ask if we decide to add different output formats, for example.

Reply:
I see. Could we support combining the options by displaying just the common columns with deprecated/vulnerable/outdated markers?

Reply:
I guess. It would mean that we would just see package + requested & resolved version though (or package + resolved version for the transitive portion).

Reply:
In the spirit of @JonDouglas's comment above (even though this scenario isn't explicitly called out), we'll go with the lowest common denominator (the current mutually exclusive experience) for now and look at adding combinations in later versions.